### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by Scylla2020** [hacksider/Deep-Live-Cam#147]
Hi I get an error during the pip install. Please assist

```
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    torch==2.0.1+cu118 from https://download.pytorch.org/whl/cu118/torch-2.0.1%2Bcu118-cp310-cp310-win_amd64.whl#sha256=f58d75619bc96e4322343c030b893613701caa2d6db8017155da226c14171335 (from -r c:\Users\UserX\OneDrive\Desktop\AI\Deep-Live-Cam\requirements.txt (line 11)):
        Expected sha256 f58d75619bc96e4322343c030b893613701caa2d6db8017155da226c14171335
             Got        5707a17926024d740f59172e6fd88e63e15366e44cfcffe9ce880b84ef14d6db
2024-08-10 05:04:58.433 [info] Traceback (most recent call last):
```

**Comment by indieshack** [comment]
Ditto

**Comment by bobmane** [comment]
Windows PowerShell
...
[notice] A new release of pip available: 22.2.1 -> 24.2
[notice] To update, run: python.exe -m pip install --upgrade pip
(venv) PS C:\Deep-Live-Cam> python.exe -m pip install --upgrade pip
Requirement already satisfied: pip in c:\deep-live-cam\venv\lib\site-packages (22.2.1)
Collecting pip
  Downloading pip-24.2-py3-none-any.whl (1.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 190.9 kB/s eta 0:00:00
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 22.2.1
    Uninstalling pip-22.2.1:
      Successfully uninstalled pip-22.2.1
Successfully installed pip-24.2
(venv) PS C:\Deep-Live-Cam> pip install -r requirements.txt
Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/cu118
Ignoring torch: markers 'sys_platform == "darwin"' don't match your environment
Ignoring torchvision: markers 'sys_platform == "darwin"' don't match your environment
Ignoring onnxruntime: markers 'sys_platform == "darwin" and platform_machine != "arm64"' don't match your environment
Ignoring onnxruntime-silicon: markers 'sys_platform == "darwin" and platform_machine == "arm64"' don't match your environment
Ignoring tensorflow: markers 'sys_platform == "darwin"' don't match your environment
Collecting numpy==1.23.5 (from -r requirements.txt (line 3))
  Downloading numpy-1.23.5-cp310-cp310-win_amd64.whl.metadata (2.3 kB)
Collecting opencv-python==4.8.1.78 (from -r requirements.txt (line 4))
  Downloading opencv_python-4.8.1.78-cp37-abi3-win_amd64.whl.metadata (20 kB)
Collecting onnx==1.16.0 (from -r requirements.txt (line 5))
  Downloading onnx-1.16.0-cp310-cp310-win_amd64.whl.metadata (16 kB)
Collecting insightface==0.7.3 (from -r requirements.txt (line 6))
  Downloading insightface-0.7.3.tar.gz (439 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting psutil==5.9.8 (from -r requirements.txt (line 7))
  Downloading psutil-5.9.8-cp37-abi3-win_amd64.whl.metadata (22 kB)
Collecting tk==0.1.0 (from -r requirements.txt (line 8))
  Downloading tk-0.1.0-py3-none-any.whl.metadata (693 bytes)
Collecting customtkinter==5.2.2 (from -r requirements.txt (line 9))
  Downloading customtkinter-5.2.2-py3-none-any.whl.metadata (677 bytes)
Collecting pillow==9.5.0 (from -r requirements.txt (line 10))
  Downloading Pillow-9.5.0-cp310-cp310-win_amd64.whl.metadata (9.7 kB)
Collecting torch==2.0.1+cu118 (from -r requirements.txt (line 11))
  Downloading https://download.pytorch.org/whl/cu118/torch-2.0.1%2Bcu118-cp310-cp310-win_amd64.whl (2619.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╺━━━━━━ 2.2/2.6 GB 163.4 kB/s eta 0:46:10
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    torch==2.0.1+cu118 from https://download.pytorch.org/whl/cu118/torch-2.0.1%2Bcu118-cp310-cp310-win_amd64.whl#sha256=f58d75619bc96e4322343c030b893613701caa2d6db8017155da226c14171335 (from -r requirements.txt (line 11)):
        Expected sha256 f58d75619bc96e4322343c030b893613701caa2d6db8017155da226c14171335
             Got        ae8f20e6388b9b8df854b5b1175502cb28b092a7d008a693d75c67b219e57000

(venv) PS C:\Deep-Live-Cam>
